src/hd/block.c

```diff
@@ -978,6 +1002,71 @@ void add_scsi_sysfs_info(hd_data_t *hd_data, hd_t *hd, char *sf_dev)
         }
       }
 
+      str_printf(&pr_str, 0, "%s model", hd->unix_dev_name);
+      PROGRESS(5, 3, pr_str);
+
+      memset(scsi_cmd_buf, 0, sizeof scsi_cmd_buf);
+      memset(&hdr, 0, sizeof(hdr));
+
+      hdr.interface_id = 'S';
+      hdr.cmd_len = 6;
+      hdr.dxfer_direction = SG_DXFER_FROM_DEV;
+      hdr.dxferp = scsi_cmd_buf + 8 + 6;
+      hdr.dxfer_len = 0x60;
+      hdr.cmdp = scsi_cmd_buf + 8;
+      hdr.cmdp[0] = 0x12;
+      hdr.cmdp[4] = 0x60;
```

Comment:
too much magic, can you please add to comment for which this constants stands for?